monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -28,17 +34,33 @@ groups:
           type: ceph_default
           oid: 1.3.6.1.4.1.50495.15.1.2.3.1
         annotations:
-          description: Monitor count in quorum is low.
+          description: |
+            Monitor count in quorum is below three.
```

Comment:
Nice! While you're at it, mind changing the alert here as well?
Something like
```
sum(ceph_mon_quorum_status) < (count(ceph_mon_quorum_status) / 2) + 1
```

Reply:
As discussed, we'll postpone and enhance this proposal to be included in another PR.